### ISSUE TITLE
Sketch quick templates missing some sizes

### ISSUE THREAD

**Issue by Ashton-W** [bjango/Bjango-Templates#2]
Thanks for the templates Marc.

I'm using the **App Icon - Apple iOS (Quick)** template and the `icon` slice doesn't have a `40w` `-40` export.
This is the _iPad Spotlight iOS 7-9 40pt 1x_ icon.

I also noticed that you need two copies (for iPad iPhone iOS 7.0 and later) of some of the sizes, `58`, `80`, and `120`. Should the slice export two copies of these for you?

**Comment by marcedwards** [comment]
> Thanks for the templates Marc.

Not a problem at all!

> I'm using the App Icon - Apple iOS (Quick) template and the icon slice doesn't have a 40w -40 export.

I actually had them in the templates, and they were removed on Jan 26 as part of https://github.com/bjango/Bjango-Templates/commit/71097c5e8e386db7844b4c285b2f8da35ade6079. This was because Apple no longer mentions the icon size in their docs:
https://developer.apple.com/ios/human-interface-guidelines/graphics/app-icon/

I guess they’ve deprecated iOS 7-9 support?

> I also noticed that you need two copies (for iPad iPhone iOS 7.0 and later) of some of the sizes, 58, 80, and 120. Should the slice export two copies of these for you?

You only need a single file for the cases where multiple instances are needed.

I initially started with `icon-60@2x` style filenames, but it got confusing, and meant having a file for `icon-60@2x` and a file for `icon-40@3x`, which gives no benefit except making your app bundle bigger. So, I then switched to just using `icon-120`.

Yep, some of these decisions are subjective. Feel free to fork or modify the templates to your liking.

I hope that helps!
